crates/services/consensus_module/poa/src/service_test/trigger_tests.rs

```diff
@@ -104,7 +104,27 @@ impl DefaultContext {
         importer
             .expect_block_stream()
             .returning(|| Box::pin(tokio_stream::pending()));
+
+        let mut block_producer = MockBlockProducer::default();
+        block_producer
+            .expect_produce_and_execute_block()
+            .returning(|_, time, _| {
+                let mut block = Block::default();
+                block.header_mut().set_time(time);
+                block.header_mut().recalculate_metadata();
+                Ok(UncommittedResult::new(
+                    ExecutionResult {
+                        block,
+                        skipped_transactions: Default::default(),
+                        tx_status: Default::default(),
+                        events: Default::default(),
+                    },
+                    Default::default(),
```

Comment:
maybe we want to implement `Default` for `ExecutionResult` and convert to the following -
```suggestion
                    ExecutionResult {
                        block,
                        ..Default::default(),
                    },
                    Default::default(),
```

Reply:
Nice suggestion, will do.

Reply:
Done in [5fed46e](https://github.com/FuelLabs/fuel-core/pull/2145/commits/5fed46e1eb06ba58face9351e99d5f5e5ea78fba)